src/content/enterprise/index.md

```diff
@@ -52,6 +52,7 @@ Some collaborative efforts to make Ethereum enterprise friendly have been made b
 ### Products and services {#products-and-services}
 
 - [Alchemy](https://www.alchemy.com/) _provides API services and tools for building and monitoring applications on Ethereum_
+- [Blast](https://blastapi.io/) _a multichain, subscription-based, API platform, featuring the fastest and most reliable blockchain access to Ethereum Archive Mainnet and Tetnets._
```

Comment:
> featuring the fastest and most reliable blockchain

Can we remove the salesy language here, please? We have no way of proving this claim.

> a multichain,

Don't think this is necessarily appropriate information for people interested in enterprise-level **Ethereum**

See other examples for more appropriate language choices here please :)

Reply:
I changed the text.

Reply:
Hey @minimalsm ,

Is there something else needed from us for the PR? We can also schedule a call for a demo regarding our Blast platform, if needed.

Reply:
Hey @minimalsm ,

Happy new year!

I just wanted to know if there is something else needed from us for the PR.